docs/repos/tfvc/workspace-command.md

```diff
@@ -114,6 +114,29 @@ If you delete a workspace that contains pending changes, TFVC cancels the pendin
 > [!NOTE]
 > Commands that run manually require the `/noprompt` option to bypass user acknowledgement. Be careful if you use the PowerShell `Start()` method to run commands. The `/noprompt` option can be automatically set in PowerShell.
 
+
+When deleting a workspace you need to provide the `<workspace-owner>` and `<workspace-name>`
```

Comment:
When you delete a workspace, you need to provide values for the `<workspace-owner>` and `<workspace-name>` parameters.